nautobot/docs/user-guide/platform-functionality/savedview.md

```diff
@@ -0,0 +1,45 @@
+# Saved Views
+
++++ 2.3.0
+
+Saved Views allow users to save their list view configurations, including table settings, filters, pagination, and sorting details, into Saved View instances.
```

Comment:
Similarly here, let's make it clear how a user can make use of a saved view once they've saved it.